tokio/Cargo.toml

```diff
@@ -57,13 +57,6 @@ timer = ["tokio-timer"]
 udp = ["tokio-udp"]
 uds = ["tokio-uds"]
 
-# This feature comes with no promise of stability. Things will
```

Comment:
Is it me or did this change from the 1.x merge break the exporting of [these items](https://github.com/tokio-rs/tokio/blob/master/tokio/src/lib.rs#L107-L117) from tokio?

Reply:
Seems like it. We will need to work on fixing this.

Reply:
I've created #1094 to track it.